packages/trackerless-network/test/utils/utils.ts

```diff
@@ -92,7 +92,7 @@ export const createStreamMessage = (
 }
 
 export const createRandomNodeId = (): DhtAddress => {
-    return getDhtAddressFromRaw(randomBytes(10))
+    return getDhtAddressFromRaw(randomBytes(20))
```

Comment:
Is this same as Kademlia ID, so could we do?
```suggestion
    return getDhtAddressFromRaw(randomBytes(KADEMLIA_ID_LENGTH_IN_BYTES))
```